index.js

```diff
@@ -46,7 +47,7 @@ function ratelimit(opts = {}) {
     const limiter = new Limiter(Object.assign({}, opts, { id }));
 
     // check limit
-    const limit = await thenify(limiter.get.bind(limiter));
+    const limit = await util.promisify(limiter.get.bind(limiter))();
```

Comment:
maybe we can try `Limit.prototype.get = util.promisify(Limit.prototype.get)` outside of the middleware function to avoid transfer it every request.